r/src/compute.cpp

```diff
@@ -325,6 +325,19 @@ std::shared_ptr<arrow::compute::FunctionOptions> make_compute_options(
         cpp11::as_cpp<arrow::TimeUnit::type>(options["unit"]));
   }
 
+  if (func_name == "strftime") {
+    using Options = arrow::compute::StrftimeOptions;
+    std::string format = "%Y-%m-%dT%H:%M:%S";
```

Comment:
As there are no default values for `format` in the C++ code, you probably don't want to set this here - the R code should either supply a value for `format` or this should fail.  See the way that the options are implemented for the `extract_regex` kernel (in this file) as an example of this.

Reply:
Done, please see if it makes sense now.